demo/src/app/pages/modules/datepicker/datepicker.page.ts

```diff
@@ -112,6 +119,13 @@ export class DatepickerPage {
                     defaultValue: "0"
                 },
                 {
+                    name: "pickerInitialView",
+                    type: "CalendarViewType",
+                    description: "Specifies the initial view for the datepicker. Options are: <code>CalendarViewType.Year</code>, " +
```

Comment:
From my understanding, because CalendarViewType is an enum
```js
export enum CalendarViewType {
    Year = 0, ... 
}
```
there are no string equivalent values to use. For eg. this is not allowed
```html
<suiDatepicker pickerInitialView="year">
```
To use string values instead of enums, a new type would need to be created similar to DatepickerMode
```js
export type DatepickerMode = "year" | "month" | "date" | "datetime" | "time";
export const DatepickerMode = {
    Year: "year" as DatepickerMode, ...
}
```
What is your preferred option?